src/backend/sycl_cuda_backend.cc

```diff
@@ -1,15 +1,29 @@
 #include "backend/sycl_backend.h"
 
-#include <cuda_runtime.h>
```

Comment:
I'm confused at this not requiring a cuda header.

Reply:
It split up the include into multiple internal cuda headers above. fixed.